src/components/image/index.tsx

```diff
@@ -121,8 +128,19 @@ export function Image({
 
   let imageComponent = null;
 
-  if (error && ErrorComponent) {
-    imageComponent = <ErrorComponent error={error} />;
+  if (isFetchingThing) {
+    if (LoadingComponent) {
```

Comment:
Should this check if LoadingComponent is undefined - that way a user could pass in `null` if they want to render nothing at all?